deps/pog/deps/fmt/CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ ExternalProject_Add(
 	SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/fmt"
 	CMAKE_ARGS
 		-DCMAKE_INSTALL_PREFIX=${FMT_INSTALL_DIR}
-		-DCMAKE_BUILD_TYPE=Release
+		-DCMAKE_BUILD_TYPE=$<IF:$<CONFIG:Debug>,Debug,Release>
```

Comment:
What is the motivation for using generator expression and not just `${CMAKE_BUILD_TYPE}` directly?

Reply:
That is of course much better, thank you!